src/config/testnet_postake_medium_curves.mlh

```diff
@@ -1,7 +1,7 @@
 [%%define ledger_depth 14]
 [%%import "/src/config/curve/medium.mlh"]
 [%%import "/src/config/coinbase/realistic.mlh"]
-[%%import "/src/config/scan_state/point2tps.mlh"]
+[%%import "/src/config/scan_state/medium.mlh"]
```

Comment:
Are these changes meant to be part of this PR?

Reply:
same